ci/ci_steps.sh

```diff
@@ -26,6 +26,10 @@ then
 else
   docker run -t -i -v "$ENVOY_BUILD_DIR":/build -v $TRAVIS_BUILD_DIR:/source \
     lyft/envoy-build:$ENVOY_BUILD_SHA /bin/bash -c "cd /source && ci/do_ci.sh $TEST_TYPE"
+  if [ "$TEST_TYPE" == "bazel.coverage" ]
+  then
+    ./ci/coverage_publish.sh
+  fi
 fi
 
 if [ "$TEST_TYPE" == "bazel.release" ]
```

Comment:
No, this is a separate set of instructions that pushes all of the docker images and verifies the docker-compose examples that is only run on the release build.